hail/src/main/scala/is/hail/expr/ir/streams/EmitStream.scala

```diff
@@ -154,7 +153,7 @@ object EmitStream {
 
     def emit(ir: IR, cb: EmitCodeBuilder, region: Value[Region] = outerRegion, env: Emit.E = env, container: Option[AggContainer] = container): IEmitCode = {
       ir.typ match {
-        case _: TStream => produce(ir, cb, outerRegion, env, container)
```

Comment:
this was a big one - it masked most of the rest of the bugs because nested stream elements were allocated in the partition region.